niceplots/utils.py

```diff
@@ -624,6 +625,167 @@ def plotColoredLine(
         return
 
 
+def plotNestedPie(
+    data,
+    colors=None,
+    alphas=None,
+    fig=None,
```

Comment:
Is there a need to pass in a `fig` object here? Does not seem to be used, only returned if created.